src/containers/Header/Search.tsx

```diff
@@ -1,13 +1,15 @@
-import { KeyboardEventHandler, useContext } from 'react'
+import { KeyboardEventHandler, useContext, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useNavigate } from 'react-router-dom'
 import { XrplClient } from 'xrpl-client'
-
 import {
   isValidClassicAddress,
   isValidXAddress,
   classicAddressToXAddress,
 } from 'ripple-address-codec'
+import PropTypes from 'prop-types'
```

Comment:
PropTypes are not needed if you have typescript.  We are trying to get rid of all uses of it.